frontend/testing/playwright/tests/data-model/data-model.spec.ts

```diff
@@ -55,46 +56,40 @@ test('Allows to add a data model, include an object with custom name and fields
     await dataModelPage.tabOutOfNameField();
     const newNameFieldValue = await dataModelPage.getNameFieldValue();
     expect(newNameFieldValue).toEqual(newName);
-    await dataModelPage.checkThatTreeItemProperyExistsOnScreen(newName);
+    await dataModelPage.checkThatTreeItemPropertyExistsOnScreen(newName);
   };
 
   // Rename the object
   const treeItemTestName: string = 'test';
   await replaceName0WithNewTextValue(treeItemTestName);
   await dataModelPage.clickOnTreeItemProperty(treeItemTestName);
 
-  // Helper function to add a new field to the test object added
-  const addFieldToTheTestNode = async () => {
+  // Helper function to add a new property to the test object added
+  const addPropertyToTheObjectNode = async (propertyName: TextKey) => {
     await dataModelPage.focusOnTreeItemProperty(treeItemTestName);
-    await dataModelPage.clickOnAddNodeToPropertyButton();
-    await dataModelPage.clickOnAddFieldToNodeButton();
+    await dataModelPage.clickOnObjectAddPropertyButton();
+    await dataModelPage.clickOnAddPropertyToObjectButton(propertyName);
   };
 
-  // Add 'text1' field to the object
-  await addFieldToTheTestNode();
+  // Add 'text' property to the object
+  await addPropertyToTheObjectNode('schema_editor.add_string');
   await replaceName0WithNewTextValue('text1');
+  expect(await dataModelPage.isTypeComboboxVisible()).toBeFalsy();
 
-  // Add 'text2' field to the object
-  await addFieldToTheTestNode();
-  await replaceName0WithNewTextValue('text2');
-
-  // Add 'number' field to the object
-  await addFieldToTheTestNode();
-  await dataModelPage.checkThatTreeItemProperyExistsOnScreen(name0);
+  // Add 'number' property to the object
+  await addPropertyToTheObjectNode('schema_editor.add_number');
```

Comment:
Then we can adjust this to:
```suggestion
  await addPropertyToTheObjectNode('number');
```